src/MiniProfiler.Shared/MiniProfiler.Settings.cs

```diff
@@ -148,6 +149,15 @@ public static class Settings
             public static bool ShowControls { get; set; } = false;
 
             /// <summary>
+            /// Custom timing ExecuteTypes to ignore as duplicates in the UI.
+            /// </summary>
+            public static HashSet<string> CustomTimingExecuteTypesToIgnoreAsDuplicate { get; } = new HashSet<string>()
```

Comment:
`CustomTimingExecuteTypesToIgnoreAsDuplicate` is a bit long, how about `IgnoredDuplicateExecuteTypes`?